smooth-scroll.py

```diff
@@ -167,14 +122,11 @@ def scroll(self, amount: int) -> None:
         else:  # inertial scroll
             self.inertial_scroll(amount, duration)
 
-        # report we are done
-        self.sender.send_cmd('set-option window scroll_running ""', client=True)
-
 
 def parse_options(option_name: str) -> dict:
     """Parse a Kakoune map option and return a str-to-str dict."""
     items = [
-        elt.split('=', maxsplit=1)
+        elt.split("=", maxsplit=1)
```

Comment:
Sorry for these minor formatting changes, I forgot to turn off my Python auto-formatting plugin.

Reply:
No worries, looked very much like `black`'s doing which I had tried to stay mostly compatible with.

Reply:
No worries, looked very much like `black`'s doing which I had originally tried to stay mostly compatible with.